ci-operator/step-registry/agent-qe/baremetal/install/ove/disconnected/mirror-utility-images/agent-qe-baremetal-install-ove-disconnected-mirror-utility-images-commands.sh

```
@@ -0,0 +1,420 @@
+        "quay.io/kubevirt/vm-killer:${KUBEVIRT_TAG}"
+        "quay.io/kubevirt/winrmcli:${KUBEVIRT_TAG}"
+        "ghcr.io/astral-sh/uv:latest"
+    )
+}
+
+# 3. Proxy Configuration
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy"
+        echo "source ${SHARED_DIR}/proxy-conf.sh"
+        source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)